app/controllers/tour_controller.rb

```diff
@@ -0,0 +1,26 @@
+class TourController < ApplicationController
```

Comment:
Add a `layout` call to the start of this controller so that you don't have to specify the layout for every action:

```ruby
class TourController < ApplicationController
  layout 'without_sidebar'

  def index; end
```